projects/plugins/jetpack/modules/widgets/eu-cookie-law/eu-cookie-law.js

```diff
@@ -10,6 +10,10 @@
 		initialScrollPosition,
 		scrollFunction;
 
+	if ( null === widget || null === overlay ) {
```

Comment:
Was this related to the bug itself, or just a way to shortcircuit if the widget isn't found at all?

Reply:
A short circuit because the rest of the code assumes those elements are present. Thanks for the review 👍